totalRP3/modules/chatframe/chatframe.lua

```diff
@@ -354,7 +354,7 @@ TRP3_API.utils.getCharacterInfoTab = getCharacterInfoTab;
 
 ---@param message string
 ---@param NPCEmoteChatColor Color
-local function detectEmoteAndOOC(message, NPCEmoteChatColor)
+local function detectEmoteAndOOC(message, isEmote, NPCEmoteChatColor)
```

Comment:
I would usually suggest adding new function parameters at the end of the list. It's a local function so I assume you caught all usages of the function so it's probably fine here.

Reply:
Yeah I was split on that. I knew there was a single usage of the function and I was thinking about the chat color possibly being nil. But given that isEmote can also be nil, I'll swap them around